Makefile

```diff
@@ -76,14 +87,15 @@ _print_vars:
 	@echo "    DEVWORKSPACE_API_VERSION=$(DEVWORKSPACE_API_VERSION)"
 
 _create_namespace:
-	kubectl create namespace $(NAMESPACE) || true
+	$(K8S_CLI) create namespace $(NAMESPACE) || true
 
 _generate_related_images_env:
 	@mkdir -p $(INTERNAL_TMP_DIR)
+	echo "export RELATED_IMAGE_devworkspace_webhook_server=$(IMG)" > $(RELATED_IMAGES_FILE)
 	cat ./config/components/manager/manager.yaml \
 		| yq -r \
 			'.spec.template.spec.containers[]?.env[] | select(.name | startswith("RELATED_IMAGE")) | "export \(.name)=\"$${\(.name):-\(.value)}\""' \
-		> $(RELATED_IMAGES_FILE)
+		>> $(RELATED_IMAGES_FILE)
```

Comment:
Won't this duplicate `export RELATED_IMAGE_devworkspace_webhook_server=$(IMG)`?

Reply:
Not really. We'll get in the end:

```
export RELATED_IMAGE_devworkspace_webhook_server=sleshchenko/che-workspace-controller:local
export ....
export RELATED_IMAGE_devworkspace_webhook_server="${RELATED_IMAGE_devworkspace_webhook_server:-quay.io/devfile/devworkspace-controller:next}"
```
I think that's should be OK for code simplification where we don't need to analyze `.spec.template.spec.containers[]?.env[].name` to exclude webhookserver.
Does it make sense?